alma/controllers/hook/GetContentHookController.php

```diff
@@ -246,6 +247,9 @@ public function processConfiguration()
             $descriptionPaymentTrigger = Tools::getValue('ALMA_DESCRIPTION_TRIGGER');
             Settings::updateValue('ALMA_DESCRIPTION_TRIGGER', $descriptionPaymentTrigger);
 
+            $activateFragment = (bool) Tools::getValue(FragmentAdminFormBuilder::ALMA_ACTIVATE_FRAGMENT . '_ON');
+            Settings::updateValue(FragmentAdminFormBuilder::ALMA_ACTIVATE_FRAGMENT, $activateFragment);
```

Comment:
You use eanble for others activation  no ?

Reply:
@Benjamin-Freoua-Alma 
Yes for my code, but it's merge code between Arnaud and Olivier :p